.travis.yml

```diff
@@ -58,11 +38,17 @@ jobs:
          - kcov --version
          - RUSTFLAGS="-C link-dead-code" cargo build --features=decode_test,quick_test --tests --verbose
          - travis_wait cargo kcov -v --coveralls --no-clean-rebuild -- --verify --exclude-pattern=$HOME/.cargo,aom_build,.h,test
+        env:
+         - CACHE_NAME=COVERALLS
       - name: "Bench"
         script: cargo bench --features=bench --no-run --verbose
+        env:
+          - CACHE_NAME=BENCH
```

Comment:
These 1 space indents all over the script are weird and it looks like I'm not alone. Maybe we should change all of them to two spaces like here

Reply:
Agreed, I wanted to avoid that within this PR because I wanted to make it clearer what I was changing. But this is going to be a big enough refactor already, maybe it makes sense to do the formatting as well.